odoo/addons/test_new_api/models/test_new_api.py

```diff
@@ -268,6 +276,13 @@ class MultiTag(models.Model):
     _description = 'Test New API Multi Tag'
 
     name = fields.Char()
+    labeled_multis = fields.One2many('test_new_api.multi', 'label')
+    labeled_count = fields.Integer(compute='_compute_labeled_count')
+
+    @api.depends('labeled_multis')
+    def _compute_labeled_count(self):
+        for record in self:
+            record.labeled_count = len(record.labeled_multis)
```

Comment:
This part looks redundant with the other test model below:
* `labeled_count` is a computed field that depends on a one2many field (`labeled_multis`) whose inverse is a computed stored many2one field (`label`) that depends on a char field (`filter`), and the test checks that modifying `filter` causes `labeled_count` to be recomputed
* `member_count` is a computed field that depends on a one2many field (`member_ids`) whose inverse is a computed stored many2one field (`container_id`) that depends on a char field (`name`), and the test checks that modifying `name` causes `member_count` to be recomputed

As the second model only needs `member_count` to be added, I would keep that second test model only.  Moreover, the other model was explicitly designed to test this kind of tricky situation.